sdk/js/src/CacheStore.ts

```diff
@@ -38,46 +23,42 @@ export abstract class CacheStore {
         return `${this.getConfigKey(user)}.fetch_date`
     }
 
-    private loadConfigUserId(user: DVCPopulatedUser): string | null | undefined {
+    private async loadConfigUserId(user: DVCPopulatedUser): Promise<string | null | undefined> {
         const userIdKey = this.getConfigUserIdKey(user)
-        return this.load<string>(userIdKey)
+        return this.store.load<string>(userIdKey)
     }
 
-    private loadConfigFetchDate(user: DVCPopulatedUser): number {
+    private async loadConfigFetchDate(user: DVCPopulatedUser): Promise<number> {
         const fetchDateKey = this.getConfigFetchDateKey(user)
-        return parseInt(this.load<string>(fetchDateKey) || '0', 10)
-    }
-
-    remove(storeKey: string): void {
-        this.store?.removeItem(storeKey)
+        return parseInt(await this.store.load<string>(fetchDateKey) || '0', 10)
```

Comment:
splitting this into two lines could make it easier to read
```suggestion
        const fetchDate = await this.store.load<string>(fetchDateKey) || '0'
        return parseInt(fetchDate, 10)
```